test/linalg/symmetric.jl

```diff
@@ -42,208 +42,265 @@ end
-        @test Hermitian(Hermitian(asym, :U))     === Hermitian(asym, :U)
-        @test Symmetric(Symmetric(asym, :U), :U) === Symmetric(asym, :U)
-        @test Hermitian(Hermitian(asym, :U), :U) === Hermitian(asym, :U)
-        @test_throws ArgumentError Symmetric(Symmetric(asym, :U), :L)
-        @test_throws ArgumentError Hermitian(Hermitian(asym, :U), :L)
-        # mixed cases with Hermitian/Symmetric
-        @test Symmetric(Hermitian(asym, :U))     === Symmetric(asym, :U)
-        @test Hermitian(Symmetric(asym, :U))     === Hermitian(asym, :U)
-        @test Symmetric(Hermitian(asym, :U), :U) === Symmetric(asym, :U)
-        @test Hermitian(Symmetric(asym, :U), :U) === Hermitian(asym, :U)
-        @test_throws ArgumentError Symmetric(Hermitian(asym, :U), :L)
-        @test_throws ArgumentError Hermitian(Symmetric(asym, :U), :L)
-
-        # similar
-        @test isa(similar(Symmetric(asym)), Symmetric{eltya})
-        @test isa(similar(Hermitian(asym)), Hermitian{eltya})
-        @test isa(similar(Symmetric(asym), Int), Symmetric{Int})
-        @test isa(similar(Hermitian(asym), Int), Hermitian{Int})
-        @test isa(similar(Symmetric(asym), (3,2)), Matrix{eltya})
-        @test isa(similar(Hermitian(asym), (3,2)), Matrix{eltya})
-        @test isa(similar(Symmetric(asym), Int, (3,2)), Matrix{Int})
-        @test isa(similar(Hermitian(asym), Int, (3,2)), Matrix{Int})
-
-        # full
-        @test asym == full(Hermitian(asym))
-
-        # parent
-        @test asym == parent(Hermitian(asym))
-
-        # getindex
-        @test asym[1,1] == Hermitian(asym)[1,1]
-        @test asym[1,1] == Symmetric(asym)[1,1]
-
-        #trace
-        @test trace(asym) == trace(Hermitian(asym))
-
-        # issymmetric, ishermitian
-        if eltya <: Real
-            @test issymmetric(Symmetric(asym))
-            @test ishermitian(Symmetric(asym))
-        end
-        if eltya <: Complex
-            @test ishermitian(Symmetric(b + b'))
-        end
+            @testset "parent" begin
+                @test asym === parent(Symmetric(asym))
+                @test aherm === parent(Hermitian(aherm))
+            end
 
-        # transpose, ctranspose
-        S = Symmetric(asym)
-        H = Hermitian(asym)
-        if eltya <: Real
-            @test  transpose(S) === S == asym
-            @test ctranspose(S) === S == asym
-            @test  transpose(H) === H == asym
-            @test ctranspose(H) === H == asym
-        else
-            @test  transpose(S) === S
-            @test ctranspose(S) ==  Symmetric(conj(asym))
-            @test  transpose(H) ==  Hermitian(transpose(asym))
-            @test ctranspose(H) === H == asym
-        end
+            @testset "getindex and unsafe_getindex" begin
+                @test aherm[1,1] == Hermitian(aherm)[1,1]
+                @test asym[1,1] == Symmetric(asym)[1,1]
+                @test Symmetric(asym)[1:2,1:2] == asym[1:2,1:2]
```

Comment:
I think when I wrote these tests, the range arguments called `unsafe_getindex`. Or I could've just been wrong 2 years ago.